packages/prerender-fargate/lib/prerender-fargate.ts

```diff
@@ -10,20 +10,63 @@ import { AccessKey, User } from 'aws-cdk-lib/aws-iam';
 import { Duration, RemovalPolicy, Stack } from 'aws-cdk-lib';
 import * as path from 'path';
 
+/**
+ * Options for configuring a Prerender Fargate service.
+ */
 export interface PrerenderOptions {
-    prerenderName: string,
-    domainName: string,
-    vpcId?: string,
-    bucketName?: string,
-    expirationDays?: number,
-    tokenList: Array<string>,
-    certificateArn: string,
-    desiredInstanceCount?: number,
-    maxInstanceCount?: number,
-    instanceCPU?: number,
-    instanceMemory?: number
-    enableRedirectCache?: string
-}
+    /**
+     * The name of the Prerender service.
+     */
+    prerenderName: string;
+    /** 
+     * The domain name to use for the Prerender service. 
+     */
+    domainName: string;
+    /** 
+     * The ID of the VPC. 
+     */
+    vpcId?: string;
+    /** 
+     * The name of the S3 bucket to use for storing Prerender data. 
+     */
+    bucketName?: string;
+    /** 
+     * The number of days to keep Prerender data before expiring it. 
+     */
+    expirationDays?: number;
+    /** 
+     * A list of tokens to use for authenticating requests to the Prerender service. 
+     */
+    tokenList: Array<string>;
+    /** 
+     * The ARN of the SSL certificate to use for the Prerender service. 
+     */
+    certificateArn: string;
+    /** 
+     * The desired number of instances in the ECS cluster. 
+     */
+    desiredInstanceCount?: number;
+    /** 
+     * The maximum number of instances in the ECS cluster. 
+     */
+    maxInstanceCount?: number;
+    /** 
+     * The amount of CPU to allocate to each instance. 
+     */
+    instanceCPU?: number;
+    /** 
+     * The amount of memory to allocate to each instance. 
+     */
+    instanceMemory?: number;
+    /** 
+     * Whether to enable caching of redirects in the Prerender service. 
+     */
+    enableRedirectCache?: string;
+    /** 
+     * Whether to create a new VPC for the Prerender service. 
+     */
+    createVpc: boolean;
```

Comment:
I don't think we need this, `vpcId` is optional so if it's not provided we should create a VPC

Reply:
This is intentional, this would ensure we won't accidentally create a VPC again for existing stacks.

Reply:
That makes sense!